databases/config/database.js

```diff
@@ -14,7 +14,8 @@ module.exports = {
     define: {
       timestamps: true,
       freezeTableName: true
-    }
+    },
+    logging: false
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change to disable logging in the database configuration could potentially make it harder to debug issues related to database operations. If you want to disable logging, consider making it configurable based on the environment or a specific flag, rather than hardcoding it to `false`.
```diff
-    logging: false
+    logging: process.env.DB_LOGGING === 'true'
```
This way, you can control the logging behavior via an environment variable.


<!-- This is an auto-generated comment by OSS CodeRabbit -->